examples/Multibus_simple/Multibus_simple.ino

```diff
@@ -0,0 +1,48 @@
+#include <OneWire.h>
+#include <DallasTemperature.h>
+
+int oneWirePins[]={3,7};//OneWire DS18x20 temperature sensors on these wires
+const int oneWirePinsCount=sizeof(oneWirePins)/sizeof(int);
+
+OneWire ds18x20[oneWirePinsCount];
```

Comment:
Oh wait, I got a bit confused there... This example is for the `DallasTemperature.h` lib... And is already merged there... However, since this change was never merged into this library, that example won't work with the official version of this library.

Another error I'm getting:

```c
Multibus_simple:23: error: 'class OneWire' has no member named 'setPin'
     ds18x20[i].setPin(oneWirePins[i]);
                ^
```